hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupMerge.java

```diff
@@ -124,4 +128,43 @@ public void TestIncBackupMergeRestore() throws Exception {
     admin.close();
     conn.close();
   }
+
+  @Test
+  public void TestIncBackupMergeRestoreSeparateFs() throws Exception {
+
+    // prepare BACKUP_ROOT_DIR on a different filesystem from HBase
+    File tempDir = new File(FileUtils.getTempDirectory(), UUID.randomUUID().toString());
+    tempDir.deleteOnExit();
+    BACKUP_ROOT_DIR = tempDir.toURI().toString();
+
+    Connection conn = ConnectionFactory.createConnection(conf1);
```

Comment:
please use try-with-resources to close the connection. Actually I'm surprised that we don't have a static analysis tool warning about this.